### PR TITLE
Add new chains for `CSUR`

### PR DESCRIPTION
**Pull request by hiletmis**
Related to #261 

Deployments shown below are mock deployments and demonstration of contract verification process.

**bsquared** [docs](https://docs.bsquared.network/for-users/connect)
---
[OwnableCallForwarder](https://bsquared.l2scan.co/address/0xef455205966486F52f7951ba4C3517239B036835) 0xef455205966486F52f7951ba4C3517239B036835 has been verified on Etherscan.
[AccessControlRegistry](https://bsquared.l2scan.co/address/0xcD7Df573B0F0bb4F2f8dFFF6650cDe8C77431730) 0xcD7Df573B0F0bb4F2f8dFFF6650cDe8C77431730 has been verified on Etherscan.
[Api3ServerV1](https://bsquared.l2scan.co/address/0x3ee097B82E270d82f829e5742cC16B7686247Ff2) 0x3ee097B82E270d82f829e5742cC16B7686247Ff2 has been verified on Etherscan.

**bsquared-testnet**
---
API N/A

**lukso** [docs](https://docs.lukso.tech/networks/mainnet/parameters)
---
[OwnableCallForwarder](https://explorer.execution.mainnet.lukso.network/address/0xef455205966486F52f7951ba4C3517239B036835) 0xef455205966486F52f7951ba4C3517239B036835 has been verified on Etherscan.
[AccessControlRegistry](https://explorer.execution.mainnet.lukso.network/address/0xcD7Df573B0F0bb4F2f8dFFF6650cDe8C77431730) 0xcD7Df573B0F0bb4F2f8dFFF6650cDe8C77431730 has been verified on Etherscan.
[Api3ServerV1](https://explorer.execution.mainnet.lukso.network/address/0x3ee097B82E270d82f829e5742cC16B7686247Ff2) 0x3ee097B82E270d82f829e5742cC16B7686247Ff2 has been verified on Etherscan.

**lukso-testnet** [docs](https://docs.lukso.tech/networks/testnet/parameters)
---
[OwnableCallForwarder](https://explorer.execution.testnet.lukso.network/address/0xef455205966486F52f7951ba4C3517239B036835) 0xef455205966486F52f7951ba4C3517239B036835 has been verified on Etherscan.
[AccessControlRegistry](https://explorer.execution.testnet.lukso.network/address/0xcD7Df573B0F0bb4F2f8dFFF6650cDe8C77431730) 0xcD7Df573B0F0bb4F2f8dFFF6650cDe8C77431730 has been verified on Etherscan.
[Api3ServerV1](https://explorer.execution.testnet.lukso.network/address/0x3ee097B82E270d82f829e5742cC16B7686247Ff2) 0x3ee097B82E270d82f829e5742cC16B7686247Ff2 has been verified on Etherscan.

polygon-sepolia-testnet [docs ](https://docs.polygon.technology/pos/reference/rpc-endpoints/?h=80002)
---
[OwnableCallForwarder](https://www.oklink.com/amoy/address/0xef455205966486F52f7951ba4C3517239B036835) at 0xef455205966486F52f7951ba4C3517239B036835
[AccessControlRegistry](https://www.oklink.com/amoy/address/0xcD7Df573B0F0bb4F2f8dFFF6650cDe8C77431730) at 0xcD7Df573B0F0bb4F2f8dFFF6650cDe8C77431730
[Api3ServerV1](https://www.oklink.com/amoy/address/0x3ee097B82E270d82f829e5742cC16B7686247Ff2) at 0x3ee097B82E270d82f829e5742cC16B7686247Ff2

sei-evm-testnet [docs](https://v2.docs.sei.io/setting-up-a-wallet)
---
Although `sei-evm-testnet` utilizes contract verification, API3 contracts verification couldn't be verified. 
Successfully verified contract Contract5 on the block explorer.
https://seitrace.com/address/0x1B127098D19A3D6A0417560fD7df2b927FafD933#code

**taiko-hekla-testnet** [docs](https://docs.taiko.xyz/network-reference/rpc-configuration)
---
[OwnableCallForwarder](https://blockscoutapi.hekla.taiko.xyz/address/0xef455205966486F52f7951ba4C3517239B036835) 0xef455205966486F52f7951ba4C3517239B036835 has been verified on Etherscan.
[AccessControlRegistry](https://blockscoutapi.hekla.taiko.xyz/address/0xcD7Df573B0F0bb4F2f8dFFF6650cDe8C77431730) 0xcD7Df573B0F0bb4F2f8dFFF6650cDe8C77431730 has been verified on Etherscan.
[Api3ServerV1](https://blockscoutapi.hekla.taiko.xyz/address/0x3ee097B82E270d82f829e5742cC16B7686247Ff2) 0x3ee097B82E270d82f829e5742cC16B7686247Ff2 has already been verified on Etherscan.

### CODE REVIEW

File: chains/sei-evm-testnet.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "alias": "sei-evm-testnet",
```
Comment (bdrhn9) [inline chains/sei-evm-testnet.json:2]: I'm not fully sure but probably I'd prefer it to be `sei-testnet`. Wdyt?

File: chains/sei-evm-testnet.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "id": "713715",
+  "name": "Sei EVM testnet",
+  "providers": [
+    {
+      "alias": "default",
+      "rpcUrl": "https://evm-rpc-arctic-1.sei-apis.com"
+    },
+    {
+      "alias": "publicnode",
```
Comment (bdrhn9) [inline chains/sei-evm-testnet.json:22]: It's not `publicnode`.

File: chains/sei-evm-testnet.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://seitrace.com/api"
+    },
+    "browserUrl": "https://seitrace.com/"
+  },
+  "id": "713715",
+  "name": "Sei EVM testnet",
```
Comment (bdrhn9) [inline chains/sei-evm-testnet.json:15]: There is also testnet for this chain which code name is `atlantic-2`. So it's better to be explicit and say `Sei EVM devnet`.

File: chains/taiko-holesky-testnet.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://blockscoutapi.hekla.taiko.xyz/api"
+    },
+    "browserUrl": "https://blockscoutapi.hekla.taiko.xyz/"
+  },
+  "id": "167009",
+  "name": "Taiko Holesky testnet",
```
Comment (bdrhn9) [inline chains/taiko-holesky-testnet.json:15]:
```suggestion
  "name": "Taiko Hekla testnet",
```

File: chains/polygon-sepolia-testnet.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  "blockTimeMs": 2211,
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://www.oklink.com/api/v5/explorer/contract/verify-source-code-plugin/amoy_testnet"
+    },
+    "browserUrl": "https://www.oklink.com/amoy"
```
[the remark this answers is not in the document]
Comment (hiletmis) [reply chains/polygon-sepolia-testnet.json:12]: Shall we replace it with polygonscan

**Comment by hiletmis** [comment]
Changes have been made 👍

**Review by bdrhn9 — APPROVED** [review]
LGTM 👍🏼  Thanks for providing documentation links, they helped me.

**Pull request merged**